### PR TITLE
Made Undo and Redo Keyboard Shortcuts Work

### PR DESCRIPTION
**Pull request by SquaredPiano**
Made Ctrl + Z undo any changes on the canvas and Ctrl + Shift + Z redo any changes done on the canvas.

### CODE REVIEW

**Review by alexapostolu — DISMISSED** [review]
lgtm!

File: src/main/java/controllers/EditorController.java [hunk 1]
```diff
@@ -142,6 +162,8 @@ public void initialize() {
 
         gc.setFill(Color.WHITE);
         gc.fillRect(0, 0, canvas.getWidth(), canvas.getHeight());
+        project.addVisualElement(new AardSquare(0, 0, Math.max(canvas.getWidth(), canvas.getHeight()),
+                true, true, Color.WHITE, Color.WHITE, 0));
```
Comment (latxh) [inline src/main/java/controllers/EditorController.java:166]: Why is this here?

Comment (latxh) [reply src/main/java/controllers/EditorController.java:166]: Discussed in discord, will approve

**Review by latxh — APPROVED** [review]
lgtm
